src/fragments/lib/analytics/android/record.mdx

```diff
@@ -101,6 +101,86 @@ RxAmplify.Analytics.flushEvents();
 </Block>
```

Comment:
Note for configuration sample above: need to verify whether the `autoFlushEventsInterval` is seconds or milliseconds. iOS sample shows seconds, Android sample shows milliseconds. That may be accurate, but wanted to double check